### DISCUSSION TITLE
Weird Ambient Weather sensor result

### DISCUSSION THREAD

**Discussion by tmarschner** [merbanan/rtl_433#2154] [Q&A]
Hi all, 

I have five Ambient Weather WH31E sensors which are being monitored with rtl_433 (integrating into Home Assistant via MQTT). I am getting the proper temperature returned for four of them but for one of I get a wacky temperature of about 400 deg F even though the display on the sensor shows room temperature (about 68 F). The Ambient Weather receiver shows the correct temperature for this sensor so it seems like the rf data isn't being decoded properly. I tried changing the sensor ID via the dip switches but the problem persists. Viewing the raw MQTT data shows also the incorrect temperature as does console output from rtl_433 (running manually in the foreground).

Does anyone have any idea what might be wrong?

Thanks,
Tom

**Comment by zuckschwerdt** [comment]
Can you run with `-R 113:vv` and show what the packet bits are (the blocks of `bitbuffer:: Number of rows: ...` outputs).
Or better grab a sample of that sensor, using `-S known` then search through the recorded files for the broken one. Upload here as zip.

**Reply by tmarschner** [reply]
Hi,

I was in the process of setting things up to collect the data and noticed that the sensor had never acquired an RCC signal so I installed a fresh set of batteries. This seems to have resolved the issue.

Thanks for your help.

Tom
